apps/ui/components/reservation/EditStep0.tsx

```diff
@@ -66,7 +67,7 @@ type Props = {
   userReservations: Node[];
   reservationForm: UseFormReturn<PendingReservationFormType>;
   activeApplicationRounds: ApplicationRoundFieldsFragment[];
-  setErrorMsg: React.Dispatch<React.SetStateAction<string | null>>;
+  errorToast: (props: ToastProps) => Id;
```

Comment:
is this needed? or can it just be imported here?

Reply:
yeah this doesn't seem necessary at all. Seems it's only used in the commented out section below - which is now removed